cmd/migration/keycloak/keycloak.go

```diff
@@ -0,0 +1,79 @@
+package keycloak
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/spf13/cobra"
+	"github.com/zitadel/zitadel-tools/internal/migration"
+)
+
+// Cmd represents the auth0 migration command
+var Cmd = &cobra.Command{
+	Use:   "keycloak",
+	Short: "Transform the exported Keycloak users and passwords to a ZITADEL import JSON",
+	RunE: func(cmd *cobra.Command, args []string) error {
+		return migrate()
+	},
+}
+
+var (
+	realmPath string
+)
+
+func init() {
+	Cmd.Flags().StringVar(&realmPath, "realm", "./realm.json", "realm export in json format")
+}
+
+func migrate() error {
+	realm, err := migration.ReadJSONFile[realm](realmPath)
+	if err != nil {
+		return fmt.Errorf("read realm: %w", err)
+	}
+	users, err := createHumanUsers(realm.Users)
+	if err != nil {
+		return err
+	}
+
+	importData := migration.CreateV1Migration(users)
+	err = migration.WriteProtoToFile(importData)
+	if err != nil {
+		return err
+	}
+	log.Println("Import file done")
+	return nil
+}
+
+// Currently ignored fields:
+// - CreatedTimestamp
+// - Enabled
+// - Totp (bool flag and credential type)
```

Comment:
I had a quick look at the import human user object in ZITADEL. there is a field otp_code, could that be used?